### ISSUE TITLE
Cannot read property 'major' of undefined

### ISSUE THREAD

**Issue by brainsiq** [tjunnone/npm-check-updates#224]
Am getting this issue on one project, which stops modules being updated:

```
Unhandled rejection TypeError: Cannot read property 'major' of undefined
    at /home/chris/.nvm/versions/node/v4.4.2/lib/node_modules/npm-check-updates/lib/version-util.js:68:71
    at Array.filter (native)
    at Object.stringify (/home/chris/.nvm/versions/node/v4.4.2/lib/node_modules/npm-check-updates/lib/version-util.js:67:10)
    at isUpgradeable (/home/chris/.nvm/versions/node/v4.4.2/lib/node_modules/npm-check-updates/lib/versionmanager.js:149:31)
    at /home/chris/.nvm/versions/node/v4.4.2/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:8201:21
    at /home/chris/.nvm/versions/node/v4.4.2/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:4299:13
    at /home/chris/.nvm/versions/node/v4.4.2/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:3073:15
    at baseForIn (/home/chris/.nvm/versions/node/v4.4.2/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:2033:14)
    at pickByCallback (/home/chris/.nvm/versions/node/v4.4.2/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:4298:7)
    at Function.<anonymous> (/home/chris/.nvm/versions/node/v4.4.2/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:9893:11)
    at Function.pick (/home/chris/.nvm/versions/node/v4.4.2/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:8152:31)
    at baseWrapperValue (/home/chris/.nvm/versions/node/v4.4.2/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:2768:30)
    at LodashWrapper.wrapperValue (/home/chris/.nvm/versions/node/v4.4.2/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:6124:14)
    at Object.upgradeDependencies (/home/chris/.nvm/versions/node/v4.4.2/lib/node_modules/npm-check-updates/lib/versionmanager.js:135:10)
    at /home/chris/.nvm/versions/node/v4.4.2/lib/node_modules/npm-check-updates/lib/npm-check-updates.js:112:39
    at tryCatcher (/home/chris/.nvm/versions/node/v4.4.2/lib/node_modules/npm-check-updates/node_modules/bluebird/js/release/util.js:16:23)
```

Dependencies:
```
 "dependencies": {
    "chai": "3.x.x",
    "glob": "7.x.x",
    "grunt": "0.x.x",
    "grunt-cli": "1.x.x",
    "grunt-contrib-clean": "x.x.x",
    "grunt-contrib-copy": "1.x.x",
    "grunt-contrib-handlebars": "0.10.x",
    "grunt-contrib-jshint": "1.x.x",
    "grunt-contrib-symlink": "1.x.x",
    "grunt-jscs": "2.x.x",
    "grunt-karma": "0.x.x",
    "i-spy": "0.x.x",
    "karma": "^0.13.19",
    "karma-chrome-launcher": "0.x.x",
    "karma-firefox-launcher": "0.x.x",
    "karma-mocha": "0.x.x",
    "karma-teamcity-reporter": "0.x.x",
    "load-grunt-tasks": "^3.3.0",
    "mocha": "2.x.x",
    "requirejs": "2.x.x"
  }
```

**Comment by metaraine** [comment]
Thank you for reporting! It may be related to the ".x.x" syntax. I haven't
seen that before. I assume it is valid semver, but would it be possible for
you to simply use ".x" as a workaround? It could also be the "x.x.x" which
can be replaced by "*".

**Comment by clubdesign** [comment]
Hey, same error here:

`
"dependencies": {
    "babel-polyfill": "*",
    "babel-runtime": "*",
    "concurrently": "^2.0.0",
    "humps": "^1.0.0",
    "immutable": "*",
    "isomorphic-fetch": "*",
    "jquery": "*",
    "lodash": "*",
    "normalize.css": "*",
    "normalizr": "*",
    "react": "^15.0.1",
    "react-dom": "^15.0.1",
    "react-redux": "*",
    "react-router": "^2.2.1",
    "react-router-redux": "^*",
    "redux": "*",
    "redux-logger": "*",
    "redux-thunk": "^2.0.1"
  },
  "devDependencies": {
    "babel-core": "*",
    "babel-loader": "*",
    "babel-plugin-transform-runtime": "*",
    "babel-preset-es2015": "*",
    "babel-preset-react": "*",
    "babel-preset-react-hmre": "*",
    "babel-preset-stage-0": "*",
    "browser-sync": "*",
    "browser-sync-webpack-plugin": "*",
    "concurrently": "*",
    "css-loader": "*",
    "express": "*",
    "extract-text-webpack-plugin": "*",
    "file-loader": "*",
    "isomorphic-style-loader": "*",
    "json-loader": "*",
    "node-sass": "*",
    "postcss": "*",
    "postcss-import": "*",
    "postcss-loader": "*",
    "postcss-scss": "*",
    "precss": "*",
    "redux-devtools": "*",
    "redux-devtools-dock-monitor": "*",
    "redux-devtools-log-monitor": "*",
    "sass-loader": "*",
    "url-loader": "*",
    "webpack": "*",
    "webpack-dev-middleware": "*",
    "webpack-hot-middleware": "*"
  }
`

**Comment by metaraine** [comment]
I confirmed that x.x.x is a valid semver range. There must be a bug in version-until which does some custom version range parsing. I can look at this when I get a chance, or someone else can work on a pull request. Should be an easy change!

**Comment by metaraine** [comment]
Fixed in 2.6.2.
